src/components/LoginForm.js

```diff
@@ -12,6 +13,7 @@ const LoginForm = () => {
     const navigate = useNavigate();
     const { notificationHandler } = useNotification();
     const { updateWatchLater } = useWatchLater();
+    // const { updateLikes} = useLikes();
```

Comment:
suggestion: remove commented code if not required

Reply:
removed.